server/template_test.go

```diff
@@ -460,7 +460,7 @@ func TestPushedCommitsTemplate(t *testing.T) {
 
 func TestCreateMessageTemplate(t *testing.T) {
 	expected := `
-[panda](https://github.com/panda) just created branch [\[mattermost-plugin-github:branch\]](https://github.com/mattermost/mattermost-plugin-github/tree/branch)
+[\[mattermost-plugin-github\]](https://github.com/mattermost/mattermost-plugin-github) branch [branch](https://github.com/mattermost/mattermost-plugin-github/tree/branch) created by [panda](https://github.com/panda)
```

Comment:
(0/5): `branch branch` seems a bit repetitive, and may cause confusion. Maybe change the branch name to something like exampleBranch?

Reply:
Good suggestion :+1: